proto/xudrpc.proto

```diff
@@ -467,6 +467,15 @@ message Order {
   uint64 hold = 10 [json_name = "hold"];
 }
 
+message Currency {
+  // Ticker symbol of this currency (e.g. BTC)
+  string ticker_symbol = 1 [json_name = "ticker_symbol"];
+  // The places to the right of the decimal point of the smallest subunit (e.g. 8, 18)
+  uint64 digits = 2 [json_name = "digits"];
```

Comment:
We call this `decimal_places` elsewhere, I'm thinking we should stick to that same name here even tho it's a bit longer.